app/src/main/java/com/dmuench/scatterhunt/MainFragment.java

```diff
@@ -261,8 +261,8 @@ public void onLocationResult(LocationResult locationResult) {
 
                     location = locationResult.getLastLocation();
                     if (location != null) {
-                        state.putDouble("latitude", location.getLatitude());
-                        state.putDouble("longitude", location.getLongitude());
+                        state.putString("latitude", Double.toString(location.getLatitude()));
```

Comment:
sending it as a double in the args was causing an error. Rather than go through the struggle of setting up SafeTypeArguments which involves an entirely separate library we chose to just convert it to a string and back.